backend/src/common/enums/app/env.enum.ts

```diff
@@ -21,6 +21,16 @@ const ENV = {
   API: {
     V1_PREFIX: '/api/v1',
   },
+  MAIL: {
```

Comment:
It is not correct.  You should provide params from .env file. 

You also commit smtp password to the public repository. Now anyone can use our mail server.
 Are you crazy?